src/DetailsView/components/overview-content/overview-help-links.tsx

```diff
@@ -2,13 +2,14 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { ExternalLink } from '../../../common/components/external-link';
+import { ExternalLink, ExternalLinkDeps } from '../../../common/components/external-link';
 import { NamedSFC } from '../../../common/react/named-sfc';
 import { HyperlinkDefinition } from '../../../views/content/content-page';
-import { HelpLinkDeps } from './overview-help-section';
+
+export type HelpLinksDeps = ExternalLinkDeps;
 
 export interface HelpLinksProps {
-    deps: HelpLinkDeps;
+    deps: HelpLinksDeps;
```

Comment:
Can we directly use `ExternalLinkDeps` here? I dont think it will matter much.

Reply:
The patter we're trying to follow is each component declare its own, direct dependencies and we use the '&' operator to bring the children's dependencies, In case the component the component doesn't have any direct dependency, we still declare the type. I guess this make for less changes if we need to add direct dependencies in the future and also it's nice to have one pattern only.

Reply:
So, I guess my question here will be then: do we need to export this type that is only used here?